action.sh

```diff
@@ -1 +1,63 @@
 #!/bin/bash
+
+set -euo pipefail ## if anything in the script fails => exit 
+
+REMOTE_USER="root"
+REMOTE_IP="68.183.145.114"
+STEPS=()
+
+
+while [[ $# -gt 0 ]]; do ## define arguments
+    case "$1" in 
+        --runs-on)
+            VM=$2
+            shift 2
+        ;;
+        --step)
+            STEPS+=("$2")
+            shift 2
+        ;;
+        *)
+            echo "Usage: action --runs-on USER@HOST --step \"command_or_script\" [--step ...]"
```

Comment:
`Usage: action --runs-on USER@HOST --step \"command_or_script\" [--step ...]`

You already specify the command --runs-on but doesn't implement